src/test/regress/output/multi_alter_table_statements.source

```diff
@@ -437,75 +441,24 @@ SELECT indexname, tablename FROM pg_indexes WHERE tablename = 'lineitem_alter';
 
 DROP INDEX temp_index_2;
 ERROR:  index "temp_index_2" does not exist
--- verify that distributed ddl commands use 2PC, even the value of citus.multi_shard_commit_protocol is '1pc'
+--- verify that distributed ddl commands use the value of citus.multi_shard_commit_protocol
```

Comment:
Description not entirely accurate anymore, maybe "verify that we can perform distributed ddl commands using 2pc." and set to 2PC for both create and drop index.